### ISSUE TITLE
Feature Request: Login as user

### ISSUE THREAD

**Issue by philmmanjaro** [kekskurse/go-mail-admin#39]
In some cases the user wants to manage his own email account with restricted rights. Based on #38 it might be able to be integrated similar to #32. This way **go-mail-admin** gains similar features in that regard as **postfixadmin** has.

**Comment by kekskurse** [comment]
Yes thas is planed, i currently need to find a good way to validate the passwort. Which actions should a user be possible to do? I currently just think about changing the password.

**Comment by philmmanjaro** [comment]
Ya, password change would be enough. On my setup I managed to accomplish that also with the **roundcube-password-plugin** and an updated SQL statement: `$config['password_query'] = "UPDATE vmail.accounts SET password=%D WHERE username=LEFT(%u, LOCATE('@', %u) - 1) AND enabled = true";`
